tests/test_wave_05.py

```diff
@@ -71,30 +69,39 @@ def test_create_goal(client):
         }
     }
 
-@pytest.mark.skip(reason="test to be completed by student")
+#@pytest.mark.skip(reason="test to be completed by student")
 def test_update_goal(client, one_goal):
-    pass
+    
     # Act
-    # ---- Complete Act Here ----
-
+    response = client.put("/goals/1", json ={
+        "title": "Updated Goal Title"
+    })
+    response_body = response.get_json()
+    
     # Assert
-    # ---- Complete Assertions Here ----
-    # assertion 1 goes here
-    # assertion 2 goes here
-    # assertion 3 goes here
+    assert response.status_code == 200
+    assert "goal" in response_body
+    assert response_body == {
+        "goal": {
+            "id": 1, 
+            "title": "Updated Goal Title"
+        }
+    }
+    goal = Goal.query.get(1)
```

Comment:
Nice work including the test that queries the database an make sure the change has been persisted.